beets/library.py

```diff
@@ -1394,28 +1394,14 @@ def parse_query_parts(parts, model_cls):
 
     # Special-case path-like queries, which are non-field queries
     # containing path separators (/).
-    path_parts = []
-    non_path_parts = []
-    for s in parts:
-        if PathQuery.is_path_query(s):
-            path_parts.append(s)
-        else:
-            non_path_parts.append(s)
+    parts = [f"path:{s}" if PathQuery.is_path_query(s) else s for s in parts]
```

Comment:
Because this looks somewhat odd, just leaving a note to myself about why we're doing this string-manipulation thing instead of constructing the `PathQuery` directly. The problem is outlined here: https://github.com/beetbox/beets/issues/1865#issuecomment-180017864